### ISSUE TITLE
question about dataset filtering

### ISSUE THREAD

**Issue by Wangchentong** [jasonkyuyim/se3_diffusion#16]
Hi,

&nbsp;&nbsp;Thank you and your team for open-sourcing this impressive work.I want to express my appreciation for your team's great work and for releasing it as open source. I'm really interested in understanding more about your **data filtering strategy.**

&nbsp;&nbsp;1. I noticed that you only trained the model on monomeric CIF files with max_len < 512. Would it be beneficial to include single chains from multimeric CIF files as well? This approach would significantly increase the dataset size by **5x to 100k proteins**, even though their structure may not be true in the monomer state.

&nbsp;&nbsp;2. Have you considered expanding the dataset using representative **clustered proteins in the high confidence AFDB**? If not, what is your opinion on this approach?

Thank you for your help and I look forward to hearing back from you soon.

**Comment by jasonkyuyim** [comment]
Hi,

Thanks for the message.

1. Yes we only trained on monomers to simplify the application of our method. A follow-up work would seek to train on multimeric chains and do more interest tasks (binder design, protein-protein interaction). Our paper was primarily concerned with the SE(3) diffusion methodology and theoretical groundings which was very non-trivial! We have found more data to lead to better results so I am optimistic more data in these regimes would help.
2. The thought of using AFDB high confidence proteins did cross my mind but was not high priority for the reasons I stated above; it was outside the scope of this work. I think it is a very interesting extension as we've seen that AF2 distillation in RFdiffusion help it tremendously. Here it will be interesting to see how a pure MSA-free generative model can learn from the same source of structures.

Happy to answer any more questions!

**Comment by jasonkyuyim** [comment]
1. I view RFdiffusion as an extension of FrameDiff where pre-trainedRosettaFold is trained with the the SE(3) diffusion derived in FrameDiff. (I was involved in incorporating SE(3) diffusion into RFdiffusion.) There are some minor differences like the rotation loss. I think additional investigation into the benefits of pre-training is an important one (as we stated in the paper). What I meant by AF2 distillation is that RosettaFold uses a form of distillation from AFDB so I think it plays a large part in why RFdiffusion works well.
3. I am personally not working on this. I think it is a great application.

Btw I'm extremely busy with end of semester matters. My responses will likely be delayed.

**Comment by jasonkyuyim** [comment]
I beleve a lightweight model like FrameDiff is better for methods research. It's easier to train and analyze rather than having to go through the RosettaFold pipeline which is not publicly trainable. Obviously larger more complex models will perform the best but uncovering the scaling principles will be done first through lightweight models.
